reference/constraints/Range.rst

```diff
@@ -394,15 +396,17 @@ minMessage
 **type**: ``string`` **default**: ``This value should be {{ limit }} or more.``
 
 The message that will be shown if the underlying value is less than the
-`min`_ option, and no `max`_ option has been defined (if both are defined, use
-`notInRangeMessage`_).
+`min`_ option, and **no** `max`_ option has been defined.
+
+If **both** are defined, use `notInRangeMessage`_ or use the parameter ``{{ min }}`` instead.
```

Comment:
The "or use the parameter ``{{ min }}`` instead" part does not look right to me.